loaders/loaders_config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "production": {
+    "CONFIG_FILE": "~/SCIENCE/taxonomy/gtdb/load_config.toml",
```

Comment:
Whatever's needed from these config files should just be moved into the config, don't reference separate config files.

Reply:
👍 make sense. updated config file.